README.md

```diff
@@ -13,16 +13,20 @@ You'll need [Ruby & Jekyll](https://jekyllrb.com/docs/installation/) to run the
 
 ## Making a Post
 
-To make a new post, make a new file in `_posts/` with a title of `YYYY-MM-DD-title-goes-here`. At the top of the file you'll want to provide the following information:
+To make a new post, run the following Ruby make command from root or the
+`_posts` directory (run `rake posts:new -- --help` to see help):
 
-```md
----
-layout: post # Always post
-title: "#<PR number> <PR title>"
----
+```shell
+rake posts:new -- --pr NUMBER --host USERNAME --date YYYY-MM-DD
 ```
 
-After that, it's just simple markdown. The site will auto-generate the rest.
+The `--date` argument is optional and defaults to next Wednesday.
+
+The command will create a new markdown file for the post, with metadata and
+initial Notes and Questions headers.
+
+You may need to run `gem install curb` the first time to install Ruby libcurl
+bindings.
```

Comment:
It's unfortunate that this might also involve installing libcurl. For example, on my ubuntu machine I needed to run `sudo apt install libcurl4-openssl-dev` before running `gem install curb`.

Reply:
For a long time, Curb had fewer dependencies and was more minimal and fast than other Ruby HTTP libraries. I haven't checked in the last 2-3 years though, so I'll have another look.

Reply:
For a simple use case like this, no HTTP client is needed. I reimplemented the HTTP GET call without Curb/libcurl or any other external lib.